docs/docs/guides/integrating-with-misk/1-admin-dashboard.mdx

```diff
@@ -0,0 +1,3 @@
+# Admin Dashboard
+
+- Review steps for adding a custom tab to dashboard (ie: `$ miskweb misk`)
```

Comment:
Not for this PR, but we should definitely have an explanation of the difference between `admin dashboard` and a `misk tab`